examples/step-22/step-22.cc

```diff
@@ -615,10 +619,12 @@ namespace Step22
 
     QGauss<dim> quadrature_formula(degree + 2);
 
-    FEValues<dim> fe_values(fe,
-                            quadrature_formula,
-                            update_values | update_quadrature_points |
-                              update_JxW_values | update_gradients);
+    FEValues<dim> fe_values(fe,                          //
```

Comment:
IMO it's more confusing that helpful, i would leave it as-is.

Reply:
I agree. It's ok as is.